sql/catalyst/src/main/scala/org/apache/spark/sql/types/Metadata.scala

```diff
@@ -229,6 +231,9 @@ class MetadataBuilder {
     this
   }
 
+  /** Puts a null. */
+  def putNull(key: String): this.type = put(key, null)
```

Comment:
The method is created for code organization and clarity purpose. This way it can be easily reused in the future.